torch/onnx/symbolic.py

```diff
@@ -90,7 +90,7 @@ def _unimplemented(op, msg):
 # increasing this number.  This includes symbolic definitions NOT in this
 # file, so grep for "OpName" (with quotes)
 
-_onnx_opset_version = 6
+_onnx_opset_version = 7
```

Comment:
I don't intend to push this, but including it in the PR to trigger test coverage of the modified code